consumption.tex

```diff
@@ -65,12 +54,12 @@ \subsection{Verifying Block Connection}\label{section:block-connection}
 
 \noindent
 \textbf{Federation.}
-A simpler approach is to allow a federation of nodes monitoring the source chain to vote for the blocks that are included in the best source chain.
+A simpler approach is to allow a federation of $n$ nodes monitoring the source chain to vote for their view of the best source chain.
 
 \import{./}{algorithms/alg.is-in-best-chain-federation.tex}
 
-Let us assume that the smart contract holds in its state the continuously updated set of all valid blocks on the best source blockchain, in the form of a Merkle Tree root $\mathcal{M}$. Then, by verifying a Merkle Tree inclusion proof \textsf{blockConnection} for $\textsf{b} \in \mathcal{M}$ a verifier can be certain that the block provided is indeed part of the best source blockchain. This completes our {\sf is-in-best-chain}$_\mathcal{M}$ function, shown in Algorithm~\ref{alg.is-in-best-chain-federation}.
+The best source chain is expressed as the root $\mathcal{M}$ of a Merkle Tree containing the chain's blocks as leaves. Each federation node calculates $\mathcal{M}$ and submits their vote for it. When a majority of $\lfloor\frac{n}{2}\rfloor + 1$ nodes agrees on the same $\mathcal{M}$, it is considered valid.
```

Comment:
You need to say that each federation node connects to both networks and recalculates $\mathcal{M}$ whenever a new source chain block is found.